peps/pep-0776.rst

```diff
@@ -0,0 +1,754 @@
+PEP: 776
+Title: Emscripten Support
+Author: Hood Chatham <roberthoodchatham at gmail.com>
+Sponsor: Łukasz Langa <lukasz at python.org>
+Discussions-To: https://discuss.python.org/t/pep-776-emscripten-support/84996
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 18-Mar-2025
+Python-Version: 3.14
+Post-History: `18-Mar-2025 <https://discuss.python.org/t/pep-776-emscripten-support/84996`__
```

Comment:
```suggestion
Post-History: `18-Mar-2025 <https://discuss.python.org/t/pep-776-emscripten-support/84996>`__
```

Reply:
PS For your next PEP, we usually prefer to have a first editing pass here and get it merged to https://peps.python.org/ before opening the discussion, so people have a single canonical version to review.